cellprofiler/gui/pipelinecontroller.py

```diff
@@ -4176,6 +4187,7 @@ def __init__(self, file_callback_fn, text_callback_fn):
 
     def OnDropFiles(self, x, y, filenames):
         self.file_callback_fn(x, y, filenames)
+        return True
 
     def OnDropText(self, x, y, text):
         self.text_callback_fn(x, y, text)
```

Comment:
Should this also return `True`?

Reply:
You mean OnDropText? I suspected it might, but I'm not sure how to properly trigger that function to test it.

Reply:
@AetherUnbound Yeah. I think this is correct: https://wxpython.org/Phoenix/docs/html/wx.TextDropTarget.html#wx.TextDropTarget.OnDropText

@DavidStirling please return `True` from `OnDropText`.